backend/app/core/auth.py

```
@@ -1,11 +1,11 @@
 import secrets
 import string
-from datetime import datetime, timedelta
+from datetime import datetime, timedelta, timezone
 from typing import Any, Optional, Union
```

Ruff (0.12.2)

`typing.Any` imported but unused

Remove unused import

(F401)